remoting/exchange_client.go

```diff
@@ -87,6 +91,21 @@ func (cl *ExchangeClient) doInit(url *common.URL) error {
 	return nil
 }
 
+// increase number of service using client
+func (client *ExchangeClient) IncreaseActiveNumber() {
+	atomic.AddUint32(&client.activeNum, 1)
```

Comment:
// increase number of service using client
func (client *ExchangeClient) IncreaseActiveNumber() uint32 {
	      return atomic.AddUint32(&client.activeNum, 1)
}

Reply:
done